### ISSUE TITLE
Installation fails for Xcode 6.3 GM

### ISSUE THREAD

**Issue by stevenp** [supermarin/Alcatraz#255]
The installer doesn't work for the Xcode 6.3 GM released today. To reproduce:

To reproduce:
1. Install Xcode 6.3 (6D570).
2. Install Alcatraz: `curl -fsSL https://raw.githubusercontent.com/supermarin/Alcatraz/master/Scripts/install.sh | sh`

The package manager is not displayed in the Window menu.

**Comment by rkumar-coursera** [comment]
Should already be fixed in master but you can also manually add the UUID using the following:

`defaults write ~/Library/Application\ Support/Developer/Shared/Xcode/Plug-ins/Alcatraz.xcplugin/Contents/Info DVTPlugInCompatibilityUUIDs -array-add 9F75337B-21B4-4ADC-B558-F9CADF7073A7`

**Comment by rajatk** [comment]
Should already be fixed in master, but you can also resolve it yourself by pasting the following in your terminal and restarting Xcode: 

`defaults write ~/Library/Application\ Support/Developer/Shared/Xcode/Plug-ins/Alcatraz.xcplugin/Contents/Info DVTPlugInCompatibilityUUIDs -array-add 9F75337B-21B4-4ADC-B558-F9CADF7073A7`

**Comment by azzoor** [comment]
If you download the latest source and build it locally it works.

**Comment by webdevotion** [comment]
Solution from @rajatk worked for me.  This does not mean the installed plugins will continue to work.  At least Alcatraz shows up now.

**Comment by jurre** [comment]
Plugins will need to add support for 6.3 themselves. We'll push a release soon.

**Comment by fuxx** [comment]
From our other discussion about the 6.3 beta issues. 

You could use the following to update *all plugins* at once. Also this command will work with future release of Xcode 6.+ as it use `defaults read` to get the current Xcode UUID.

```
find ~/Library/Application\ Support/Developer/Shared/Xcode/Plug-ins -name Info.plist -maxdepth 3 | xargs -I{} defaults write {} DVTPlugInCompatibilityUUIDs -array-add `defaults read /Applications/Xcode.app/Contents/Info DVTPlugInCompatibilityUUID`
```

**Comment by paulofierro** [comment]
Thanks @fuxx, works like a charm :+1:

**Comment by JackieQi** [comment]
@rajatk @fuxx Solutions work well.

**Comment by supermarin** [comment]
Fixed in 1.1.2 release. Thanks @davelyon !

**Comment by Ziewvater** [comment]
I tried most of the things described above (uninstall, reinstall, adding the new UUID to compatibility list), but Alcatraz didn't show up until I restarted my computer. Kinda smelly, but it worked after that so ¯\_(ツ)_/¯
